docker-services.yml

```diff
@@ -105,3 +105,13 @@ services:
     environment:
       - "OPENSEARCH_HOSTS=http://search:9200"
       - "DISABLE_SECURITY_DASHBOARDS_PLUGIN=true"
+  iipserver:
+    image: registry.cern.ch/docker.io/iipsrv/iipsrv:latest
```

Comment:
Wondering if we should pin to version 1.2.

Reply:
For the OpenShift image, we'll be pinning to specific versions, but for local dev it's a good chance to possibly catch breaking changes.